DXRando/DeusEx/Classes/LevelInfo.uc

```diff
@@ -20,4 +18,6 @@ function SpawnScript()
         dxr = Spawn(class'DXRando');
         dxr.SetdxInfo(Self);
     }
+
+    _SpawnScript();
```

Comment:
just moving this down so dxr.SetdxInfo(Self); gets called before the mission script is created, because we're hijacking the mission number for backtracking